### DISCUSSION TITLE
Trying to test a locally build docker BASE and RADI image.

### DISCUSSION THREAD

**Discussion by Bhardwaj-Himanshu** [JdeRobot/RoboticsAcademy#2035] [Q&A] [answered]
I recently created a docker image for an issue, for which I built `BASE and RADI IMAGES` using `./build.sh -f test` commands inside scripts directory.

I then try to install requirements.txt using `python3 -m pip install -r requirements.txt` to install neccessary dependencies mentioned in the project, but it always ends up on this error,
```bash
ERROR: Could not find a version that satisfies the requirement catkin-pkg-modules==0.5.2 (from versions: 66.0.0, 66.0.2)
ERROR: No matching distribution found for catkin-pkg-modules==0.5.2
```

For which I tried opening the requirements.txt in a text editor and set the version to 66.0.2 for catkin-pkg-modules but it then throws error as-
`dependencies till 0.5.2 not found`

What am I doing wrong here?
Doing some research on it, found out it is related to Catkin package modules or ROS.

**Comment by jmplaza** [comment, marked as answer]
Hi @Bhardwaj-Himanshu ,

we have recently updated the instructions for developers of RoboticsAcademy. Take a look at them at https://github.com/JdeRobot/RoboticsAcademy/blob/master/docs/generate_a_mini_radi.md

And feel free to use the RoboticsAcademy/scripts/mini_RADI/build.sh script, which makes it easy.

**Reply by codezerro** [reply]
Generate RADI link updated, here it is:
https://github.com/JdeRobot/RoboticsAcademy/blob/humble-devel/docs/generate_a_radi.md
